config/cloudsql-postgres.yaml

```diff
@@ -0,0 +1,19 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Ensure to replace with the right host configuration.
+database: cloudsqlpostgres
```

Comment:
Just noticing that this name is defined by gorm: https://gorm.io/docs/connecting_to_the_database.html (no action needed)

Reply:
Yes I was trying to stay consistent so we can reuse them as DriverName

Reply:
I agree, that seems good. Thanks!